examples/styled/README.md

```diff
@@ -0,0 +1,122 @@
+# Styled
+
+This example shows how to provide styles to the image focal point component.
+
+## How to use
+
+Import the library styles and the component. Use it with minimal properties:
+
+```jsx
+import '@lemoncode/react-image-focal-point/style.css';
+import { ImageFocalPoint } from '@lemoncode/react-image-focal-point';
+
+const App = () => {
+  return (
+    <ImageFocalPoint
+       src="your-image-src"
+       onChange={(newFocalPoint) => {
+         // Whatever you want to do when the user clicks on the image
+       }}
+     />
+  );
+}
+
+```
+
+Create a css file and provide some styles to the root component:
+
+_app.css_
+
+```css
+.image-focal-point {
+  width: 500px;
+  height: 500px;
+}
+
+```
+
+Use it:
+
+```diff
+import '@lemoncode/react-image-focal-point/style.css';
+import { ImageFocalPoint } from '@lemoncode/react-image-focal-point';
++ import './app.css';
+
+const App = () => {
+  return (
+    <ImageFocalPoint
++      className="image-focal-point"
+       src="your-image-src"
+       onChange={(newFocalPoint) => {
+         // Whatever you want to do when the user clicks on the image
+       }}
+     />
+  );
+}
+
+```
+
+You can also provide styles to the image and the focal point:
```

Comment:
You can also customize the look & feel of the the image and the focal point by adding custom styles

Reply:
Done